### PR TITLE
Add Styles in JS RFC

### PR DESCRIPTION
**Pull request by klzns**

### CODE REVIEW

File: text/0002-styles-in-js.md [hunk 1]
```diff
@@ -0,0 +1,367 @@
+- Start Date: 2020-03-31
+- Implementation PR: (leave this empty)
+- Store Discussion Issue: (leave this empty)
+
+# Summary
+
+This RFC proposes a solution to use the values defined in VTEX Tachyons in JavaScript-land.
```
Comment (lbebber) [inline text/0002-styles-in-js.md:7]:
```suggestion
This RFC proposes a solution to use the values defined in VTEX Tachyons both in user-land CSS and in JavaScript-land.
```

ou "user-defined CSS", se nao quiser repetir "land"

Comment (lbebber) [reply text/0002-styles-in-js.md:7]: esquece, vi agora q é só sobre js

**Review by lbebber — APPROVED** [review]

File: text/0002-styles-in-js.md [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+// Spacing API
+const space = getSpacing(2) // 0.25
+
+// Border Width API
+const border = getBorderWidth(1) // 0.125
+```
+
+# Motivation
+
+Apps like [vtex.device-detector](https://github.com/vtex-apps/device-detector) uses some [hard-coded values](https://github.com/vtex-apps/device-detector/blob/86076b0886bac7ddeada03625d2035407b19834f/react/useDevice.tsx#L19-L23) for breakpoints. With this new API those values would be the ones in effect in the store.
```
Comment (Klynger) [inline text/0002-styles-in-js.md:40]: I don't think is a good idea to add the lines in the link, is so easy to break this link

Comment (klzns) [reply text/0002-styles-in-js.md:40]: GitHub has a feature of getting a permalink, this will never change.

Comment (lbebber) [reply text/0002-styles-in-js.md:40]: @Klynger i think, but I might be mistaken, that this link refers to the commit specifically. it should only break if the commit gets removed, which shouldn't happen if we are linking to a commit in master, no?

Comment (Klynger) [reply text/0002-styles-in-js.md:40]: Oh, nice then.

**Review by Klynger — APPROVED** [review]

**Pull request merged**